invenio_app_rdm/records_ui/views/__init__.py

```diff
@@ -79,11 +80,17 @@ def create_blueprint(app):
         routes["dashboard_home"],
         view_func=dashboard,
     )
+
     blueprint.add_url_rule(
         routes["dashboard_item"],
         view_func=dashboard,
     )
 
+    blueprint.add_url_rule(
```

Comment:
This is not needed as the `routes["dashboard_item"]` handles the dashboard request.